### ISSUE TITLE
feelpp + clang-11 + boost1.71: qs_hdg_laplacian_2d failed to compile (debian bullseye)

### ISSUE THREAD

**Issue by Trophime** [feelpp/feelpp#1635]
For the records, latest feelpp dev FTBS on debian Bullseye with errors in quickstart laplacian hdg:
```
make[2]: *** [feelpp/quickstart/laplacian/CMakeFiles/feelpp_qs_hdg_laplacian_2d.dir/build.make:82: feelpp/quickstart/laplacian/CMakeFiles/feelpp_qs_hdg_laplacian_2d.dir/qs_hdg_laplacian.cpp.o] Error 1
make[2]: Target 'feelpp/quickstart/laplacian/CMakeFiles/feelpp_qs_hdg_laplacian_2d.dir/build' not remade because of errors.
make[1]: *** [CMakeFiles/Makefile2:42231: feelpp/quickstart/laplacian/CMakeFiles/feelpp_qs_hdg_laplacian_2d.dir/all] Error 2
Scanning dependencies of target feelpp_qs_laplacian_2dh
[ 95%] Building CXX object feelpp/quickstart/laplacian/CMakeFiles/feelpp_qs_laplacian_2dh.dir/qs_laplacian.cpp.o
In file included from /home/LNCMI-G/trophime/github/feelpp/feelpp/quickstart/laplacian/qs_laplacian.cpp:25:
In file included from /home/LNCMI-G/trophime/github/feelpp/feelpp/feel/../feel/feelcore/environment.hpp:46:
In file included from /home/LNCMI-G/trophime/github/feelpp/feelpp/contrib/pybind11/include/pybind11/pybind11.h:43:
In file included from /home/LNCMI-G/trophime/github/feelpp/feelpp/contrib/pybind11/include/pybind11/attr.h:13:
In file included from /home/LNCMI-G/trophime/github/feelpp/feelpp/contrib/pybind11/include/pybind11/cast.h:16:
/home/LNCMI-G/trophime/github/feelpp/feelpp/contrib/pybind11/include/pybind11/detail/internals.h:186:9: warning: 'PyEval_InitThreads' is deprecated [-Wdeprecated-declarations]
        PyEval_InitThreads();
        ^
/usr/include/python3.9/ceval.h:130:1: note: 'PyEval_InitThreads' has been explicitly marked deprecated here
Py_DEPRECATED(3.9) PyAPI_FUNC(void) PyEval_InitThreads(void);
^
/usr/include/python3.9/pyport.h:508:54: note: expanded from macro 'Py_DEPRECATED'
#define Py_DEPRECATED(VERSION_UNUSED) __attribute__((__deprecated__))
                                                     ^
/home/LNCMI-G/trophime/github/feelpp/feelpp/quickstart/laplacian/qs_hdg_laplacian.cpp:172:50: error: use of undeclared identifier 'f'
        auto u = cgLaplacian( cgXh, std::tuple{K,f,p_exact,un,r_1,r_2} );
                                                 ^
/home/LNCMI-G/trophime/github/feelpp/feelpp/quickstart/laplacian/qs_hdg_laplacian.cpp:172:60: error: use of undeclared identifier 'un'
        auto u = cgLaplacian( cgXh, std::tuple{K,f,p_exact,un,r_1,r_2} );
                                                           ^
/home/LNCMI-G/trophime/github/feelpp/feelpp/quickstart/laplacian/qs_hdg_laplacian.cpp:172:63: error: use of undeclared identifier 'r_1'; did you mean '_1'?
        auto u = cgLaplacian( cgXh, std::tuple{K,f,p_exact,un,r_1,r_2} );
                                                              ^~~
                                                              _1
/usr/include/boost/bind/placeholders.hpp:46:38: note: '_1' declared here
BOOST_STATIC_CONSTEXPR boost::arg<1> _1;
                                     ^
/home/LNCMI-G/trophime/github/feelpp/feelpp/quickstart/laplacian/qs_hdg_laplacian.cpp:172:67: error: use of undeclared identifier 'r_2'; did you mean '_2'?
        auto u = cgLaplacian( cgXh, std::tuple{K,f,p_exact,un,r_1,r_2} );
                                                                  ^~~
                                                                  _2
/usr/include/boost/bind/placeholders.hpp:47:38: note: '_2' declared here
BOOST_STATIC_CONSTEXPR boost::arg<2> _2;
                                     ^
/home/LNCMI-G/trophime/github/feelpp/feelpp/quickstart/laplacian/qs_hdg_laplacian.cpp:177:49: error: use of undeclared identifier 'locals'
                                        _inputs=locals
                                                ^
/home/LNCMI-G/trophime/github/feelpp/feelpp/quickstart/laplacian/qs_hdg_laplacian.cpp:208:33: error: use of undeclared identifier 'f'
                          _expr=f*id(w), _quad=ioption("rhs_quad") );
                                ^
/home/LNCMI-G/trophime/github/feelpp/feelpp/quickstart/laplacian/qs_hdg_laplacian.cpp:211:39: error: use of undeclared identifier 'un'
                          _expr=id(l)*un, _quad=ioption("rhs_quad")  );
                                      ^
/home/LNCMI-G/trophime/github/feelpp/feelpp/quickstart/laplacian/qs_hdg_laplacian.cpp:215:40: error: use of undeclared identifier 'r_2'; did you mean '_2'?
                           _expr=id(l)*r_2);
                                       ^~~
                                       _2
```

Note also the warning about python 3.9 pybind11. I guess this could be solved using the latest pybind11 release (2.6.2).

**Comment by romainhild** [comment]
It is probably because you don't have sympy : 
https://github.com/feelpp/feelpp/blob/0dd6b3f9563c8a52458ea72d6b4459c85cd31bab/feelpp/quickstart/laplacian/qs_hdg_laplacian.cpp#L106

**Comment by Trophime** [comment]
Ok I'll check.
But I don't remember to have seen something about sympy during the cmake...

**Comment by prudhomm** [comment]
`apt install python3-sympy` 

if you install the feelpp debian packages python3-sympy is installed

**Comment by Trophime** [comment]
Even after adding python3-sympy, I still get some errors

[make.log](https://github.com/feelpp/feelpp/files/6686696/make.log)

Note that I haven't tried yet to use fix #1635 (just load the latest dev rev)
